src/bundle/Controller/InvalidateTokenController.php

```diff
@@ -38,7 +38,7 @@ class InvalidateTokenController
      * @param int $ttl
      * @param \FOS\HttpCache\ResponseTagger $tagHandler
      *
-     * @internal param string $invalidatetoken
+     * {@internal param string $invalidatetoken}
```

Comment:
If the goal was to keep but hide an old @param, it could have been:
```suggestion
     * {@internal @param string $invalidatetoken}
```
If there is no reseon to keep this trace, it could just be removed.

Reply:
Full removal in c304af64aa06807f1b467d4b09be358a28bb54f1